### PR TITLE
Added logo to readme

### PR DESCRIPTION
**Pull request by mashrin**

### CODE REVIEW

**Comment by mashrin** [comment]
Removed the heading (Comiko and replaced with the logo) - The text along with the logo was looking a bit repetitive.

**Review by kimond — CHANGES_REQUESTED** [review]
As mentioned in the issue #50. It would be better to place the logo like [most](https://github.com/avajs/ava) [_sindresorhus_](https://github.com/sindresorhus) [repos](https://github.com/sindresorhus/xo).

**Comment by GabLeRoux** [comment]
But what about [my dark theme](https://userstyles.org/styles/37035/github-dark)!?

image

It's not centered and your logo seems badly scaled :P

**Comment by mashrin** [comment]
Am I allowed to edit the image and remove the height because there's a lot of white space?

**Comment by mashrin** [comment]
image
Looks something like this as the image is having a lot of whitespace:
image

**Comment by kimond** [comment]
@mashrin Yeah you can edit the image in order to remove the extra whitespace.

**Comment by emileber** [comment]
This fixes #50

**Comment by mashrin** [comment]
@kimond Done #53

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
+# Comiko
 
+<img src="lib/assets/comiko_long.png" alt = "Comiko" width="1153">
```
Comment (emileber) [inline README.md]: In order to align the logo correctly, you should embed it inside another tag:

```
<h1 align="center">
	<br>
	<img width="475" src="https://raw.githubusercontent.com/comiko-app/comiko/master/lib/assets/comiko.png" alt="Comiko">
	<br>
	<br>
	<br>
</h1>
```

File: README.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
+# Comiko
```
Comment (emileber) [inline README.md]: You can remove the title, you were right, it's repetitive.

**Review by emileber — CHANGES_REQUESTED** [review]
Thanks for the contribution! Last little changes and it'll be perfect.

**Comment by GabLeRoux** [comment]
@kimond you should extract from a vector and use the exact right resolution. It's blurry on my side ;)

image

**Review by emileber — APPROVED** [review]
Good enough for now, I propose we merge this and change to the new vectorized logo later.

**Review by kimond — APPROVED** [review]

**Pull request merged**